connector/avro/pom.xml

```diff
@@ -70,12 +70,10 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-tags_${scala.binary.version}</artifactId>
     </dependency>
-    <!-- #if scala-2.13 -->
```

Comment:
Are we sure about deleting these comments? I was hesitant before, because I didn't know if we would need to use a similar approach to dynamically modify dependencies when supporting Scala 3, so I took a conservative approach and kept them.

Reply:
Do we need to change `scala-parallel-collections` for Scala 3 upgrade? I would think we'd need it for other libraries. I am not removing the mechanism.

Reply:
It seems that for Scala 3 the `scala.binary.version` is always `3`? Then it looks like we can delete this comment, the corresponding dependency for Scala 3 is

```
<dependency>
    <groupId>org.scala-lang.modules</groupId>
    <artifactId>scala-parallel-collections_3</artifactId>
    <version>1.0.4</version>
</dependency>
```
It seems that this dependency no longer needs to be dynamically changed.